django/template/loaders/base.py

```diff
@@ -44,3 +93,10 @@ def reset(self):
         templates or cached loader modules).
         """
         pass
+
+    @property
+    def supports_recursion(self):
+        """
+        Used during deprecation period of non-recursive loaders.
```

Comment:
Since ``supports_recursion`` is called whenever extending happens, adding a ``DeprecatedInDjango21Warning`` to ``supports_recursion`` breaks a large number of Django tests, not all related to templates. Is it sufficient to add a note to ``internals/deprecations.txt``? Or else add the warning filter to ``runtests.py``?